docs/docs/guides/build/components/building-pipelines-with-components/using-template-udfs.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Use template variables to inject user-defined functions into component definitions.
+sidebar_position: 350
+title: Using template UDFs 
+---
+
+Template variables can be used to create user-defined functions (UDFs). There is no formal concept of a UDF in dagster. It is just a convention to a describe template variable that is a function.
```

Comment:
Would it be accurate to say "While there is no formal concept of a user-defined function (UDF) in Dagster, you can use a template variable that is a function to accomplish the same thing"?